src/lib.rs

```diff
@@ -178,13 +178,13 @@ impl Options {
     /// ```
     ///   use dogstatsd::Options;
     ///
-    ///   let options = Options::new("127.0.0.1:9000", "127.0.0.1:9001", "", vec!(String::new()), None, None);
+    ///   let options = Options::new("127.0.0.1:9000", "127.0.0.1:9001", "", vec![(String::new(), String::new())], None, None);
     /// ```
     pub fn new(
         from_addr: &str,
         to_addr: &str,
         namespace: &str,
-        default_tags: Vec<String>,
+        default_tags: Vec<(String, String)>,
```

Comment:
I don't like making a breaking change like this. Can we maybe keep `Vec<String>` and split to extract the key? I would say this is up to @mcasper to decide, but I would prefer not breaking compatibility

Reply:
Ok, let's wait for @mcasper to decide it :)

Reply:
I agree, I think it'd be best not to make a breaking change here when we don't need to. That will also keep it in line with the tags parameters in all of the metrics methods